app/services/proofing/resolution/progressive_proofer.rb

```diff
@@ -121,7 +121,7 @@ def proof_id_address_with_lexis_nexis_if_needed
       end
 
       def should_proof_state_id_with_aamva?
-        return false unless aamva_supports_state_id_jurisdiction?
+        return false unless aamva_supports_state_id_jurisdiction? && !state_in_maintenance_window?
```

Comment:
This is very likely not what we want to do.

In fact, my intention writing this PR was to just log whether a failed call was in an expected window or not so we could tweak the alarm, but that ended up being complicated and this was easy. This is mostly just to spark conversation about what _should_ happen.